packages/cdk-app-bucket/lib/index.ts

```diff
@@ -9,17 +9,30 @@ export interface AppBucketProps {
     dynamicEnvName: string;
     projectName: string;
     sharedServicesAccountId?: string;
+    removeBucket?: string;
 }
 export class AppBucket extends cdk.Construct {
     constructor(scope: cdk.Construct, id: string, props: AppBucketProps) {
         super(scope, id);
-
-        const bucket = new s3.Bucket(this, 'Bucket', {
-            bucketName: `${props.dynamicEnvName}-${props.appName}.${props.stage}.${props.projectName}`,
-            versioned: true,
-            objectOwnership: s3.ObjectOwnership.BUCKET_OWNER_PREFERRED,
-            blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL
-        });
+        let bucket;
+
+        if(props.removeBucket == 'true'){
```

Comment:
This should be === instead of ==
And should be if(props.removeBucket) if we switched it to an optional Boolean